base/deprecated.jl

```diff
@@ -539,6 +539,63 @@ end
 
 export MathConst, @math_const
 
+# 11551
+function utf16_is_surrogate(chr::UInt16)
```

Comment:
I found other places, added the deprecations for `is_utf8_start` & `is_utf8_continuation`,
and I think it sets a good precedent for people to add unit tests for deprecations.